src/MeetingsSDKAdapter.js

```diff
@@ -951,7 +967,7 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
       this.getMeetingObservables[ID] = getMeetingWithEvents$.pipe(
         publishReplay(1),
         refCount(),
-        takeUntil(end$),
+        takeWhile((meeting) => meeting.state !== MeetingState.LEFT, true),
```

Comment:
While we don't expect to get here in our flow, you should include a check for falsy state

```suggestion
        takeWhile((meeting) =>  meeting.state && meeting.state !== MeetingState.LEFT, true),
```

Reply:
done